packages/js/product-editor/src/components/variations-table/variation-actions-menu/variation-actions-menu.tsx

```diff
@@ -45,6 +45,7 @@ export function VariationActionsMenu( {
 					>
 						<MenuItem
 							href={ selection.permalink }
+							target="_blank"
```

Comment:
When adding `target="_blank"` we should also add `rel="noreferrer"` for security reasons.

Reply:
Good call @mdperez86! I added that in the commit [2723566](https://github.com/woocommerce/woocommerce/pull/40414/commits/2723566ecb6b6a74749a98dce706b930cdad34ff).
Could you take another look at this PR?